quill-core/src/main/scala/io/getquill/ast/StatelessTransformer.scala

```diff
@@ -17,6 +19,9 @@ trait StatelessTransformer {
       case If(a, b, c)                 => If(apply(a), apply(b), apply(c))
 
       case e: Dynamic                  => e
+
+      case e @ (_: Block | _: Val) =>
```

Comment:
The transformers should work with all AST types, why not support `Block` and `Val`?